src/components/Modal/Translate.vue

```diff
@@ -20,26 +20,44 @@
   -->
 
 <template>
-	<NcModal @close="$emit('close')">
+	<NcModal size="large" @close="$emit('close')">
 		<div class="translate-dialog">
 			<h2>{{ t('text', 'Translate') }}</h2>
-			<textarea v-model="input" />
-			<div class="language-selector">
-				<label for="fromLanguage">{{ t('text', 'Source language') }}</label>
-				<NcSelect v-model="fromLanguage"
-					input-id="fromLanguage"
-					:options="fromLanguages"
-					:append-to-body="false" />
+			<em>{{ t('text', 'You can translate parts of the document by selecting the text before using the translate function.') }}</em>
+			<div class="wrapper">
+				<div class="col">
+					<div class="language-selector">
+						<label for="fromLanguage">{{ t('text', 'Translate from') }}</label>
+						<NcSelect v-model="fromLanguage"
+							input-id="fromLanguage"
+							:placeholder="t('text', 'Select language')"
+							:options="fromLanguages"
+							:disabled="disableFromLanguageSelect"
+							:append-to-body="false" />
+					</div>
+					<textarea ref="input"
+						v-model="input"
+						autofocus="true"
+						@input="autosize"
+						@focus="onInputFocus" />
+				</div>
+				<div class="col">
+					<div class="language-selector">
+						<label for="toLanguage">{{ t('text', 'Translate to') }}</label>
```

Comment:
How about changing this label to "to" only. Not sure whether this is acceptable from accessibility perspective, but it would read way nicer "Translate from <language> to <language>". @nimishavijay @marcoambrosini

Reply:
It would definitely read nicer on desktop. However, more than a11y I am concerned about translation to other languages, and it may be confusing on mobile as well, what do you think @marcoambrosini ?

Reply:
I think that it's fine to have just from and to since the heading of the dialog already says translate